app/components/avatar-upload.js

```diff
@@ -0,0 +1,26 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+    
+    actions: {
+        previewImage() {
+            let preview = document.querySelector('img');
+            let file = document.querySelector('input[type=file]').files[0];
+            let reader = new FileReader();
+         
+            reader.addEventListener("load", () => {
+           
+            preview.src = reader.result;
+            }, false);
+        
+            if (file) {
+              reader.readAsDataURL(file);
+            }
+
+            
+            this.get('member-profile').set('image',preview.src)
```

Comment:
We're setting the image value before preview.src is being set. Check the `load` event listener.

Reply:
Done. As discussed, setting of image value will be handled while saving the form data. So, I'm removing this code here.